Backport of:

From 1697553b33407927669f173dfd8c815cf0f113e1 Mon Sep 17 00:00:00 2001
From: dirk <dirk@git.imagemagick.org>
Date: Mon, 11 May 2015 11:02:38 +0000
Subject: [PATCH] Jpeg images no longer have pixels per inch as a default value
 for density units when the density is not set.

Debian-bug: https://bugs.debian.org/763799
Applied-upstream: 6.9.1.3
---
 coders/jpeg.c | 2 --
 1 file changed, 2 deletions(-)

Index: imagemagick-6.6.9.7/coders/jpeg.c
===================================================================
--- imagemagick-6.6.9.7.orig/coders/jpeg.c	2016-11-15 13:46:48.207839119 -0500
+++ imagemagick-6.6.9.7/coders/jpeg.c	2016-11-15 13:46:48.203839070 -0500
@@ -1795,7 +1795,6 @@
   else
     if (sizeof(JSAMPLE) > 1)
       jpeg_info.data_precision=12;
-  jpeg_info.density_unit=(UINT8) 1;
   if (image->debug != MagickFalse)
     (void) LogMagickEvent(CoderEvent,GetMagickModule(),
       "Image resolution: %.20g,%.20g",floor(image->x_resolution+0.5),
